tests/test_openjdk.py

```diff
@@ -82,6 +84,51 @@ def test_jdk_version(container, java_version):
     )
 
 
+def test_java_home(auto_container: ContainerData):
+    """Check that the environment variable ``JAVA_HOME`` is set to the correct
+    value.
+
```

Comment:
```suggestion
Regression test following https://bugzilla.suse.com/show_bug.cgi?id=1206128
```

I would add the bsc for reference.